services/QuillLMS/app/assets/stylesheets/pages/premium_hub/filterable-reports.scss

```diff
@@ -41,6 +41,19 @@
       box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.10);
       padding: 16px 16px 32px;
       overflow: visible;
+      .timeframe-filter {
+          &.disabled {
+            .dropdown {
+              .dropdown__control {
+                .dropdown__value-container {
+                  .dropdown__single-value {
+                    color: #7f7f7f;
+                  }
+                }
+              }
+            }
+          }
+        }
     }
```

Comment:
good call! I'll update. I already flagged the hex code to Jack so that he can map it to a variable so just waiting for that (it's used quite a bit around the codebase)